### DISCUSSION TITLE
Can I set the ouput part to be a sliding window?

### DISCUSSION THREAD

**Discussion by Excespo** [microsoft/vscode-jupyter#9863] [Questions and Answers] [answered]
Can I set the ouput part to be a sliding window just like the classic jupyter can be? In that way I can move less while see more data. Thanks!

**Comment by greazer** [comment, marked as answer]
Hi @Excespo, I assume you're wishing to be able have cell output that is scrollable separately from the notebook. Notebooks in VS Code do not support this model at the moment. However, you can change the number of lines output to be less than the default of 30, and then view the full output in a separate editor. 

Note that the full output editor is live, so it will update as more output is generated.

Here's a contrived example where I set the size limit to 5.
![Recording 2022-05-02 at 11 22 47](https://user-images.githubusercontent.com/11685408/166303507-be4c14a7-2bbf-487a-a66f-5c28854d30b5.gif)

If this isn't sufficient, please consider upvoting these two issues:
https://github.com/microsoft/vscode/issues/144111
https://github.com/microsoft/vscode-jupyter/issues/7096
